clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -66,6 +66,7 @@ class CheckedRegionFinder : public RecursiveASTVisitor<CheckedRegionFinder>
     bool VisitVarDecl(VarDecl *VD);
     bool VisitParmVarDecl(ParmVarDecl *PVD);
     bool VisitMemberExpr(MemberExpr *E);
+    bool VisitDeclRefExpr(DeclRefExpr*);
```

Comment:
Have you added a testcase for this change? I didn't see an expression that is declaring a variable.

Reply:
It's my understanding that these `DeclRef`'s appear in the tree when variables are used in expressions: 
`````c
int f(void) {
  int x;

  if (1) {
    return x + 1;
  } else {
    return 0;
  }

}
``````
Has the AST:
![image](https://user-images.githubusercontent.com/5495888/87887220-2eeff300-c9f1-11ea-840d-986c3597b908.png)

Reply:
So you are saying that every place you refer to a variable it's basically going to be a `DeclRef` ?

Reply:
That's my understanding. Also true for function calls.